src/fusor/models.py

```diff
@@ -986,10 +986,12 @@ def save_fusions_cache(
         """Save a list of translated fusions as a cache
 
         :param fusions_list: A list of FUSOR-translated fusions
-        :param output_dir: The location to store the cached file
+        :param output_dir: The location to store the cached file. If this parameter is
+            not supplied, it will default to creating a `data` directory under
+            `src/fusor`
         :param cache_name: The name for the resultant cached file
         """
-        if not cache_dir:
+        if not cache_dir or not Path.is_dir(cache_dir):
```

Comment:
See previous comment for reason why we're removing `not cache_dir`

```suggestion
if not Path.is_dir(cache_dir)
````